### ISSUE TITLE
When contributions are unavailable, link the website

### ISSUE THREAD

**Issue by kewisch** [thunderbird/thunderbird-android#8430]
When contributions are unavailable, it shows the text we devised. It does not however hyperlink thunderbird.net nor click to link to the website.

Also, there is a period missing at the end.

<img width="429" alt="image" src="https://github.com/user-attachments/assets/02877776-f8d5-4d07-ab72-073efe1dba86">

**Comment by marcRDZ** [comment]
Hi @kewisch,
If I understood properly the text 'www.thunderbird.net' lacks of hiperlink to the site and also it needs the appearance of a clickable link., but what do you mean with the period missing at the end? Is it hidden on the view or not added on string resources?
It seems easy to solve at first glance, can I work on it? 
Await your instructions ;)

**Comment by marcRDZ** [comment]
Sorry but I misunderstood what you mean by period (slot of time vs programming operator) 😅 
I'm about to release a small fix on this issue hoping it'll be right!

**Comment by kewisch** [comment]
@marcRDZ thanks for picking this up and sorry I missed your message. Indeed I meant the punctuation mark at the end of the line, and you understood correctly that the hyperlink is missing. We did something similar in https://github.com/thunderbird/thunderbird-android/commit/c777ad55ed82e69f1c0a6ee423621f891a60e52e if you want some inspiration, in that case we decided to just make the whole area clickable.

By the way, if you want to join us to chat about development do find us at https://matrix.to/#/#tb-android-dev:mozilla.org on Matrix. It is great to see that you're picking up more issues!
